src/main/features/response/views/evidence.njk

```diff
@@ -22,20 +26,46 @@
     <select name="rows[{{ index }}][type]" id="rows[{{ index }}][type]"
       class="form-control-select {% if errorType %}form-control-error{% endif %}" >
 
-      <option value="">-- Please make a selection --</option>
+      <option value="">{{ t('-- Please make a selection --') }}</option>
       {% for row in allEvidenceTypes %}
         <option value="{{ row.value }}" {% if typeValue == row.value %}selected{% endif %}>{{ row.displayValue }}</option>
       {% endfor %}
     </select>
-    <div class="panel panel-border-narrow">
-      {{ textArea(
-            label = 'Describe this evidence in more detail (optional).',
-            name = 'rows[' + index + '][description]',
-            form = form,
-            inputClass='form-control timeline-column-text',
-            rows=2
-         )
-      }}
+    <div class="panel panel-border-narrow {% if not typeValue %} js-hidden {% endif %}">
+      {% set name = 'rows[' + index + '][description]' %}
+      {% set valueDescription = form.valueFor(name) %}
+      {% set errorDescription = form.errorFor(name) %}
+      <div class="form-group {% if errorDescription %} form-group-error {% endif %}">
+        <label for="{{ name }}" id="{{ name }}[label]" class="form-label">
+          {{ t('Describe this evidence in more detail (optional).') }}
+
+          <span class="evidence-message evidence-message-CONTRACTS_AND_AGREEMENTS {% if typeValue !== EvidenceType.CONTRACTS_AND_AGREEMENTS.value %}visually-hidden{% endif %}">
```

Comment:
should these be visually-hidden or js-hidden?

Reply:
```
{% if typeValue !== EvidenceType.CONTRACTS_AND_AGREEMENTS.value %}visually-hidden{% endif %}
```